modules/publicize/ui.php

```diff
@@ -102,53 +105,55 @@ function admin_page() {
-		  		$service_num = 0;?>
-
-		  		<div class='left'>
-
-		  		<?php
-		  		foreach ( $services as $name => $service ) :
-		  			$connect_url = $this->publicize->connect_url( $name );
-		  			if ( $service_num == ( round ( ( $total_num_of_services / 2 ), 0 ) ) )
-		  				echo "</div><div class='right'>";
-		  			$service_num++;
-		  			?>
-		  			<div class="publicize-service-entry" <?php if ( $service_num > 0 ): ?>class="connected"<?php endif; ?> >
-			  			<div id="<?php echo esc_attr( $name ); ?>" class="publicize-service-left">
-			  				<a href="<?php echo esc_url( $connect_url ); ?>" id="service-link-<?php echo esc_attr( $name ); ?>" target="_top"><?php echo $this->publicize->get_service_label( $name ); ?></a>
-			  			</div>
-
-
-			  			<div class="publicize-service-right">
-			  				<?php if ( $this->publicize->is_enabled( $name ) && $connections = $this->publicize->get_connections( $name ) ) : ?>
-			  					<ul>
-				  					<?php
+		<form action="" id="publicize-form">
+			<h3 id="publicize"><?php _e( 'Publicize', 'jetpack' ) ?></h3>
+
+			<?php
+				if ( ! empty( $_GET['action'] ) && 'deny' == $_GET['action'] ) {
+					$this->denied_notice();
+				}
+			?>
+
+			<p>
+				<?php esc_html_e( 'Connect your blog to popular social networking sites and automatically share new posts with your friends.', 'jetpack' ) ?>
+				<?php esc_html_e( 'You can make a connection for just yourself or for all users on your blog. Shared connections are marked with the (Shared) text.', 'jetpack' ); ?>
+			</p>
+
+			<?php
+			if ( $this->in_jetpack ) {
+				$doc_link = "http://jetpack.me/support/publicize/";
+			} else {
+				$doc_link = "http://en.support.wordpress.com/publicize/";
+			}
+			?>
+
+			<p>&rarr; <a href="<?php echo esc_url( $doc_link ); ?>" target="_blank"><?php esc_html_e( 'More information on using Publicize.', 'jetpack' ); ?></a></p>
+
+			<div id="publicize-services-block">
+				<?php
+				$services = $this->publicize->get_services( 'all' );
+				$total_num_of_services = count ( $services );
+				$service_num = 0;?>
+
+				<div class='left'>
+
+				<?php
+				foreach ( $services as $name => $service ) :
+					$connect_url = $this->publicize->connect_url( $name );
+					if ( $service_num == ( round ( ( $total_num_of_services / 2 ), 0 ) ) )
+						echo "</div><div class='right'>";
+					$service_num++;
+					?>
+					<div class="publicize-service-entry" <?php if ( $service_num > 0 ): ?>class="connected"<?php endif; ?> >
+						<div id="<?php echo esc_attr( $name ); ?>" class="publicize-service-left">
+							<a href="<?php echo esc_url( $connect_url ); ?>" id="service-link-<?php echo esc_attr( $name ); ?>" target="_top"><?php echo $this->publicize->get_service_label( $name ); ?></a>
+						</div>
+
+
+						<div class="publicize-service-right">
+							<?php if ( $this->publicize->is_enabled( $name ) && $connections = $this->publicize->get_connections( $name ) ) : ?>
+								<ul>
+									<?php
```

Comment:
Were there any actual changes here? I'm not sure why Git is flagging these lines.